packages/api-explorer/src/block-types/Content.jsx

```diff
@@ -1,119 +1,9 @@
-
-const Loop = ({ content, column, flags }) => {
-  const elements = content.map((block, key) => {
-    const props = { key, block, flags };
-    switch (block.type) {
-      case 'textarea':
-        // eslint-disable-next-line react/no-array-index-key
-        return <TextArea {...props} />;
-      case 'html':
-        // eslint-disable-next-line react/no-array-index-key
-        return <Html {...props} />;
-      case 'embed':
-        // eslint-disable-next-line react/no-array-index-key
-        return <Embed {...props} />;
-      case 'api-header':
-        // eslint-disable-next-line react/no-array-index-key
-        return <ApiHeader {...props} />;
-      case 'code':
-        // eslint-disable-next-line react/no-array-index-key
-        return <Code {...props} dark={column === 'right'} />;
-      case 'callout':
-        // eslint-disable-next-line react/no-array-index-key
-        return <CallOut {...props} />;
-      case 'parameters':
-        // eslint-disable-next-line react/no-array-index-key
-        return <Parameters {...props} />;
-      case 'image':
-        // eslint-disable-next-line react/no-array-index-key
-        return <ImageBlock {...props} />;
-      default:
-        return null;
-    }
-  });
-  return <div>{elements}</div>;
-};
-
-const Content = props => {
-  const { body, isThreeColumn } = props;
-  const content = parseBlocks(body);
-
-  const left = [];
-  const right = [];
-  content.forEach(elem => {
-    if (elem.sidebar) {
-      right.push(elem);
-    } else {
-      left.push(elem);
-    }
-  });
-
-  if (isThreeColumn === true) {
-    return (
-      <div className="hub-reference-section">
-        <div className="hub-reference-left">
-          <div className="content-body">
-            <Loop content={left} column="left" flags={props.flags} />
-          </div>
-        </div>
-        <div className="hub-reference-right">
-          <div className="content-body">
-            <Loop content={right} column="right" flags={props.flags} />
-          </div>
-        </div>
-      </div>
-    );
-  }
-
-  return (
-    <Loop
-      content={isThreeColumn === 'left' ? left : right}
-      flags={props.flags}
-      column={isThreeColumn}
-    />
-  );
-};
-
-Loop.propTypes = {
-  content: PropTypes.arrayOf(
-    PropTypes.shape({
-      type: PropTypes.string.isRequired,
-    }),
-  ).isRequired,
-  column: PropTypes.string,
-  flags: PropTypes.shape({
-    correctnewlines: PropTypes.bool,
-  }).isRequired,
-};
-
-Loop.defaultProps = {
-  column: 'left',
-  flags: {},
-};
-
-Content.propTypes = {
-  isThreeColumn: PropTypes.oneOfType([PropTypes.bool, PropTypes.string]),
-  body: PropTypes.string,
-  flags: PropTypes.shape({}),
-};
-
-Content.defaultProps = {
-  isThreeColumn: true,
-  body: '',
-  flags: {},
-};
-
-module.exports = Content;
+// const rdmd = require('@readme/markdown');
+const rdmd = require('api-explorer/packages/markdown/dist/main');
```

Comment:
Lets call this `markdown`.

Did you *have* to do this `dist/main` thing in order to use it? Or was this before we had a webpack config in the markdown package?

Reply:
See [this comment](https://github.com/readmeio/api-explorer/pull/306/files/cad9c2472fbf15ec66cee440e29d826c506ecd7c#r335599980), above.